cli.js

```diff
@@ -1926,6 +1926,60 @@ program
         console.log();
     });
 
+program
```

Comment:
Do we want to allow someone passing the full JSON with the tenant info in there as a short hand version in alternative to passing individual properties through parameters? E.g. 

```
{
  "channels": [
    "SiteGenesis",
    "OtherSite"
  ],
  "clientId": "client-id1",
  "ecomSite": "SiteGenesis",
  "ecomTenant": "test_prd",
  "name": "ShopperClient1",
  "privateClient": true,
  "redirectUri": "http://localhost:9010/callback",
  "scopes": [
    "sfcc.products",
    "sfcc.catalogs",
    "sfcc.products.ro"
  ],
  "secret": "secret12345"
}
```

Reply:
I agree, here I have the flat list, but on client:add I reference a file. I think we should allow both approaches on both APIS.